### PR TITLE
Feature: project housekeeping

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
Some Housekeeping and set up **tslint**

### CODE REVIEW

**Comment by AdrieanKhisbe** [comment]
@sipi Test seems to be broken for the same reason than the other: Some issue with the demote command

File: src/org-folding-and-outline-provider.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ enum ChunkType {
     SECTION = SymbolKind.Constant,
     BLOCK = SymbolKind.Number
 }
-type Chunk = { type: ChunkType, title: string, level: number, startLine: number };
```
Comment (sipi) [inline src/org-folding-and-outline-provider.ts:20]: Is it a recommended practice in TypeScript? 
Prefixing interface with "I" is not known as best practive in Java but I does not known for TS. Although this page <https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines> claims that is not TS guidelines but "_These are Coding Guidelines for Contributors to TypeScript_" but they suggest to "_Do not use "I" as a prefix for interface names."_. 

**Perhaps, we can embrace theses coding guidelines to have a clear guidelines reference?**

File: README.md [hunk 2]
```diff
@@ -15,51 +16,51 @@ We welcome issues, feature requests, and contributions to the [repo](https://git
 
 Create outlines:
 
-![outlines](img/screenshots/01_outlines.gif)
+![outlines](images/screenshots/01_outlines.gif)
```
Comment (sipi) [inline README.md:19]: Is it necessary?
I found folder names like "src", "img", "js" quite standard. And, if we rename "img" into "images" why not renaming "src" into "sources", "docs" into "documents" ?

I vote to keep "img".

File: package.json [hunk 3]
```diff
@@ -2,10 +2,15 @@
...
+    "keywords": [
+        "org",
+        "orgmode",
+        "org-mode",
+        "emacs"
+    ],
     "version": "1.1.0-SNAPSHOT",
     "publisher": "vscode-org-mode",
-    "license": "SEE LICENSE IN LICENSE.txt",
+    "license": "GPL-3.0-or-later",
```
Comment (sipi) [inline package.json:13]: :+1:

**Review by sipi — CHANGES_REQUESTED** [review]
Ok, except I vote to keep "img" folder unless you can argue.

File: README.md (hunk 2 above)
Comment (AdrieanKhisbe) [reply README.md:19]: We had both. `img` and `images`.
I tend to prefer the former.

For some reason I tend to like `src`, `images` and `documention`. But that personal taste

File: src/org-folding-and-outline-provider.ts (hunk 1 above)
Comment (AdrieanKhisbe) [reply src/org-folding-and-outline-provider.ts:20]: About the I, this is a lint rule: `ERROR: src/org-folding-and-outline-provider.ts:20:11 - interface name must start with a capitalized I` if I don't do it.
It's then probably a rule of `tslint:latest`
But we can customize it.

(not that the guidelines you refer are about the core of typescript)

Comment (sipi) [reply src/org-folding-and-outline-provider.ts:20]: Yes, I know the referred guidelines is for the core of typescript. If it is the default configuration of tslint, I'm ok to follow this practice.

**Review by sipi — APPROVED** [review]

File: src/org-folding-and-outline-provider.ts (hunk 1 above)
Comment (AdrieanKhisbe) [reply src/org-folding-and-outline-provider.ts:20]: (I meant note rather than not :))

I'm open, still new to typescript hence not many preferences :)

**Comment by AdrieanKhisbe** [comment]
@sipi Shall I go ahead or wait for a second validation? :)

File: src/org-folding-and-outline-provider.ts (hunk 1 above)
Comment (sipi) [reply src/org-folding-and-outline-provider.ts:20]: I well understood "note" :)
I'm in favor of keeping the default tslint configuration

**Comment by sipi** [comment]
You can go ahead.

**Comment by AdrieanKhisbe** [comment]
Done =) :house_with_garden:

**Comment by sipi** [comment]
nice :)

**Pull request merged**